src/bundle/Resources/public/js/scripts/sidebar/main.menu.js

```diff
@@ -0,0 +1,132 @@
+(function(global, doc, localStorage, eZ) {
+    const HEADER_HEIGHT = 72;
+    const RESIZER_WIDTH = 10;
+    const TOGGLER_HEIGHT = 48;
+    const SECOND_LEVEL_COLLAPSED_WIDTH = 48;
+    const SECOND_LEVEL_EXPANDED_WIDTH = 220;
+    const SECOND_LEVEL_MANUAL_RESIZE_MIN_WIDTH = 80;
+    const mainMenuNode = doc.querySelector('.ibexa-main-menu');
+
+    if (!mainMenuNode) {
+        return;
+    }
+
+    const firstLevelMenuNode = mainMenuNode.querySelector('.ibexa-main-menu__navbar--first-level');
+    const secondLevelMenuNode = mainMenuNode.querySelector('.ibexa-main-menu__navbar--second-level');
+    let resizeStartPositionX = 0;
+    let secondMenuLevelCurrentWidth = secondLevelMenuNode.getBoundingClientRect().width;
+    const showSecondLevelMenu = (event) => {
+        const { toggle } = event.currentTarget.dataset;
```

Comment:
Imho there's no need to put it into new variable if you use it only in one place (in if, in this case)

Reply:
So you can remove it now ;)